opencti-platform/opencti-front/src/private/components/observations/StixCyberObservables.tsx

```diff
@@ -77,14 +76,21 @@ const StixCyberObservables: FunctionComponent = () => {
   Record<string, StixCyberObservableLine_node$data>
   >({});
 
-  const convertedFilters = convertFilters(
-    viewStorage.filters,
-  ) as unknown as Filters;
+  const finalType = () => {
```

Comment:
Can you remove the function and make it only a const ?

Reply:
If it is not a function it could be added to the paginationMethod and prevent the exportFilters